.devcontainer/readme.md

```diff
@@ -0,0 +1,24 @@
+## DevContainers
+
+[DevContainers](https://code.visualstudio.com/docs/remote/containers) lets you use a Docker container as a full-featured development environment. It allows you to open any folder inside (or mounted into) a container and take advantage of Visual Studio Code's full feature set.
+
+In this project the devcontainer starts two containers:
+1) the dev container with .net SDK
+2) a container with the Azure CosmosDB emulator
+
+The two containers are wired so after switching to the remote container, just hit 'debug/run' (cosmos version) and start debugging.
+
+### Prerequisites
+
+Visual Studio Code
+
+### Usage
+
+1. Install the 'Visual Studio Code Remote - Containers' extension.
+2. vs code will ask to open the folder inside a container. allow it.
+3. Your dev env is ready for use.
+
+### Running / Debugging FHIR
+
+1. Run the selected profile
```

Comment:
>Run the selected profile [](http://example.com/codeflow?start=3&length=24)

Is it worth to add sample how to run specific profile (R4 or STU3)

Reply:
added